packages/core/src/hooks/useSupportConfig.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 Spotify AB
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { useApi, configApiRef } from '@backstage/core-api';
+
+export type SupportItemLink = {
+  url: string;
+  title: string;
+};
+
+export type SupportItem = {
+  title: string;
+  icon?: string;
+  links?: SupportItemLink[];
+};
+
+export type SupportConfig = {
+  url: string;
+  items: SupportItem[];
+};
+
+export function useSupportConfig(): SupportConfig {
+  const config = useApi(configApiRef);
+  const supportConfig = config.getOptional('app.support') as SupportConfig;
+
+  return {
+    url: supportConfig?.url ?? 'https://github.com/backstage/backstage/issues',
+    items: supportConfig?.items ?? [
+      {
+        title: 'Support Not Configured',
+        icon: 'warning',
+        links: [
+          {
+            title: 'How to configure support info',
+            // TODO: Create a docs page / section and update link
+            url: 'https://backstage.io/docs/link/to/suport/config',
```

Comment:
```suggestion
            url: 'https://backstage.io/docs/link/to/support/config',
```

:)

maybe there is something we can do that doesn't go to a 404? just the discord or whatever

Reply:
I can add a basic docs page as part of this PR. I didn't see a natural spot, but is there somewhere existing that would be appropriate to slot a section or new page? Also happy to change to discord or something else.

Reply:
You know what? Maybe it wouldn't be too weird if the default scaffolded app (and the demo app of course) had a `/support` route per default, just a placeholder thing, that was defined directly in the app. Kinda like the catalog is, essentially. It could say something basic like welcome to your conf.appName instance, here is where you can tell your users where to reach out for general support needs.

And then your default support config could point to that and rely on it existing at the start.

Reply:
We talked about this and maybe it's better to do what you said - to make a microsite docs page later about how to actually set this up.

Reply:
Actually that's a good point. Having a built in /support route might be a good idea for this sort of thing. Wondering if it might make sense to tackle that as a separate PR? If so, I could just point to the example app `app-config.yaml` for now.

Reply:
@freben - I've updated this to point to the `app-config.yaml` in the main branch for now with a todo to add a docs page.